client/me/concierge/book/info-step.js

```diff
@@ -66,11 +70,22 @@ class InfoStep extends Component {
 	}
 
 	render() {
-		const { signupForm: { firstname, lastname, message, timezone }, translate } = this.props;
+		const {
+			currentUserLocale,
+			signupForm: { firstname, lastname, message, timezone },
+			translate,
+		} = this.props;
+		const language = getLanguage( currentUserLocale ).name;
+		const notice = ! isDefaultLocale( currentUserLocale )
```

Comment:
One unfortunate side-effect of using `isDefaultLocale` is that `en` and `en-gb` are considered different languages, and you end up with circumstances like this:

<img width="762" alt="screen shot 2018-03-08 at 4 59 54 pm" src="https://user-images.githubusercontent.com/518059/37181374-4c0da17e-22f2-11e8-8742-71662827daa5.png">

Let's check explicitly for `en` and `en-gb` here instead. I know `isDefaultLocale` makes this more reusable, but if Calypso switches to another default language our HEs are still only offering these sessions in English.

Reply:
Good point, updated.